### DISCUSSION TITLE
label not passed in case of sign and verify

### DISCUSSION THREAD

**Discussion by sujaygkulkarni-nxp** [OpenSC/OpenSC#3050] [Q&A] [answered]
The label is passed as NULL in case of sign and verify

https://github.com/OpenSC/OpenSC/blob/de5942f23a38aab773da45571291b22c9916ec6a/src/tools/pkcs11-tool.c#L1353

Could you please clarify this ?

**Comment by Jakuje** [comment, marked as answer]
The sign/verify operation identifies keys only by ID. If you want to improve the pkcs11-tool to allow selection also by the label, contributions are welcomed.

**Reply by sujaygkulkarni-nxp** [reply]
Thank you for the response.

**Reply by Jakuje** [reply]
Did this solve your issue or does it need some more clarification? If not, please mark this as answer so it wont stick here as unanswered.

**Comment by sujaygkulkarni-nxp** [comment]
answered.
